app/utils/review.js

```diff
@@ -51,26 +61,30 @@ const handlePositiveEvent = () => Alert.alert(
 );
 
 const tryReview = async() => {
-	const reviewData = await RNUserDefaults.objectForKey(reviewKey) || {};
+	const data = await AsyncStorage.getItem(reviewKey) || '{}';
+	const reviewData = JSON.parse(data);
 	const { lastReview = 0, doneReview = false } = reviewData;
 	const lastReviewDate = new Date(lastReview);
 
-	// if ask me later was pressed, we only can show again after {{numberOfDays}} days
-	// if review wasn't done yet or wasn't cancelled
+	// if ask me later was pressed earlier, we can ask for review only after {{numberOfDays}} days
+	// if there's no review and it wasn't dismissed by the user
 	if (daysBetween(lastReviewDate, new Date()) >= numberOfDays && !doneReview) {
-		setTimeout(handlePositiveEvent, popupDelay);
+		setTimeout(askReview, reviewDelay);
 	}
 };
 
-class ReviewTheApp {
+class ReviewApp {
 	positiveEventCount = 0;
 
-	pushEvent = () => {
+	pushPositiveEvent = () => {
+		if (this.positiveEventCount >= numberOfPositiveEvent) {
```

Comment:
```suggestion
		if (this.positiveEventCount > numberOfPositiveEvent) {
```

`>=` would never call `tryReview`